views.go

```diff
@@ -365,6 +365,15 @@ func (lk *logkeeper) viewBuild(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
+	if r.FormValue("metadata") == "true" {
+		payload := struct {
+			model.Build
```

Comment:
if this were a pointer to a build would the resulting JSON look different?

Reply:
Nope! The encoder ends up using reflection to get the underlying struct when it is a pointer. Pointers are really only useful when you want to omitempty with null values.